pkg/rmb/rmb.go

```diff
@@ -374,7 +381,7 @@ func newRedisPool(address string) (*redis.Pool, error) {
 
 			return nil
 		},
-		MaxActive:   5,
+		MaxActive:   int(poolSize),
```

Comment:
if MaxActive is int, then there is no need to make argument a `uint32` can also be `...int` instead 🤷🏼